cloudsql/postgresql/.gcloudignore

```diff
@@ -0,0 +1,17 @@
+# This file specifies files that are *not* uploaded to Google Cloud Platform
```

Comment:
I think this file should not be uploaded because `gcloud` generates it.

Reply:
Removed.